app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -125,6 +125,7 @@ def update_enrollment_status(enrollment, response)
     case response['status']
     when IPP_STATUS_PASSED
       if SUPPORTED_ID_TYPES.include?(response['primaryIdType'])
+        enrollment.profile.activate
```

Comment:
Should we generate a disavowal event when we do this, [similar to the GPO flow](https://github.com/18F/identity-idp/blob/929d86811b64a32f016a1d31b4b38d76d66a4990/app/controllers/idv/gpo_verify_controller.rb#L32-L38)? I think this is really a question for product so I'm going to ask it in slack as well

Reply:
Changing the topic but: to test this I went through the IPP flow locally in order to get an enrollment in the pending state. Then in the rails console I found my enrollment and called `enrollment.profile.activate` and also updated the enrollment's status to `:passed`. Then I ran the OIDC sinatra app and attempted to authenticate at IAL2 with my user. I was directed to the data-sharing page and then allowed to login

It's not the most life-like test considering I didn't run this code. But it validates that this line is the missing step. I wasn't sure how much work it'd be to get the job running locally with mock responses from the USPS for a more life-like test and figure we can (and need to, tomorrow) test it more thoroughly in dev